release-team/role-handbooks/release-notes/README.md

```diff
@@ -105,6 +105,7 @@ Release Notes lead should be responsible for granting team members required acce
 1. Add OWNERS file into release notes directory. [Sample PR](https://github.com/kubernetes/sig-release/pull/1943) for v1.25 release.
 2. Add release notes team members for CHANGELOG review/approval.
    [Sample PR](https://github.com/kubernetes/kubernetes/pull/112045) for v1.25 release.
+3. Add release notes team members to the Kubernetes org as a GitHub team [Sample PR](https://github.com/kubernetes/org/pull/3735)
```

Comment:
nit: I think a link to the file is also helpful
```suggestion
3. Add release notes team members to the `release-team-release-notes` GitHub team under `release-team` in [k/org/config/kubernetes/sig-release/teams.yaml](https://github.com/kubernetes/org/blob/main/config/kubernetes/sig-release/teams.yaml). [Sample PR](https://github.com/kubernetes/org/pull/3735)
```

Reply:
+1 to @leonardpahlke 's suggestion to state at the start of the release.
I didn't include the Line number because it changes